recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "sqlglot" %}
-{% set version = "25.10.0" %}
+{% set version = "26.3.8" %}
```

Comment:
FYI, It looks like the maintainers' release cycles are fast https://github.com/tobymao/sqlglot/tags